src/modules/index.js

```diff
@@ -0,0 +1,132 @@
+const containerElement = document.querySelector('.container-todo');
+const ulElement = document.querySelector('.ul-element');
+
+const Tasks = class {
+  constructor(description, completed = false, index = 2) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  populateFields = () => {
+    let i = -1;
+    this.tasks.forEach((task) => {
+      i += 1;
+      task.index = i;
+    });
+    localStorage.setItem('savedTasks', JSON.stringify(this.tasks));
+  };
+
+  removeTask(task) {
+    const result = this.tasks.filter((b) => b !== task);
+    this.tasks = result;
+    this.populateFields();
+  }
+
+  addTask = (newTask) => {
+    this.tasks.push(newTask);
+    this.populateFields();
+    this.displayTasks();
+  };
```

Comment:
- [ ] Kindly fix add task method currently I am not able to add a task

Reply:
Do you like me to set some tasks? or do you prefer an empty task